packages/components/src/highlight-cards/count-comparison-card.tsx

```diff
@@ -55,21 +68,47 @@ export function TrendComparison( { count, previousCount }: TrendComparisonProps
 				'highlight-card-difference--negative': difference > 0,
 			} ) }
 		>
-			<span className="highlight-card-difference-icon" title={ String( difference ) }>
+			<span className="highlight-card-difference-icon">
 				{ difference < 0 && <Icon size={ 18 } icon={ arrowDown } /> }
 				{ difference > 0 && <Icon size={ 18 } icon={ arrowUp } /> }
 			</span>
-			<span className="highlight-card-difference-absolute-value">
-				{ differenceMagnitude <= 9999 && formattedNumber( differenceMagnitude ) }
-				{ differenceMagnitude > 9999 && <ShortenedNumber value={ differenceMagnitude } /> }
-			</span>
 			{ percentage !== null && (
-				<span className="highlight-card-difference-absolute-percentage"> ({ percentage }%)</span>
+				<span className="highlight-card-difference-absolute-percentage"> { percentage }%</span>
 			) }
 		</span>
 	);
 }
 
+function TooltipContent( { count, previousCount, icon, heading }: CountComparisonCardProps ) {
+	if ( previousCount ) {
+		const difference = subtract( count, previousCount ) as number;
+		return (
+			<div className="highlight-card-tooltip-content">
+				<div className="highlight-card-tooltip-counts">
+					{ formatNumber( count, false ) }
+					{ '  ' }
+					{ difference !== 0 && difference !== null && (
+						<span className="highlight-card-tooltip-count-difference">
+							({ difference < 0 ? '-' : '+' }
+							{ formatNumber( Math.abs( difference as number ), false ) })
```

Comment:
It seems formatNumber can handle the sign with option `signdisplay`. Maybe we could leverage that.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat/NumberFormat#signdisplay

Reply:
Agreed, let's tackle this in a follow-up PR.